doc/DESI_TARGET/SCND_DIR/PHASE/outdata/VERSION/OBSCON/BITNAME.rst

```diff
@@ -0,0 +1,99 @@
+:Naming Convention: ``BITNAME.fits``,
+   where ``BITNAME`` matches the name of the secondary targeting bit used by the desitarget
+   pipeline for a given survey phase (see the desitarget GitHub repository for, `e.g.`
+   the `sv1`_ or `main`_ secondary target bitmasks).
+:Regex: ``[a-zA-Z]+\.fits``
+:File Type: FITS, 20 KB - 284 MB
+
+Contents (FITS version; see notes, below, for text version)
+===========================================================
+
+====== ======= ======== ============
+Number EXTNAME Type     Contents
+====== ======= ======== ============
+HDU0_  PRIMARY IMAGE    Empty
+HDU1_  TARGETS BINTABLE Output secondary targets
+====== ======= ======== ============
+
+FITS Header Units
+=================
+
+HDU0
+----
+
+EXTNAME = PRIMARY
+
+This HDU has no non-standard required keywords.
+
+Empty HDU.
+
+HDU1
+----
+
+EXTNAME = TARGETS
+
+Output secondary targets
+
+Required Header Keywords
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+======== ============= ===== ==================================
+KEY      Example Value Type  Comment
+======== ============= ===== ==================================
+NAXIS1   89            int   width of table in bytes
+NAXIS2   4838          int   number of rows in table
+SURVEY   "main"        str   svX for SV, main for Main Survey
+PRIMDIR  "/global/"    str   location of directory of information about corresponding primary targets
+SEP      1.0           float matching radius that was used to find primary targets (arcsec)
+MASKED   T             bool  ``True`` if targets were masked to avoid bright sources
+MASKDIR  "masks/"      str   location of directory of masks used to avoid bright sources
+SCNDDIR  "/global/"    str   directory from which secondary targets were read
+OBSCON   "DARK"        str   observing layer for file
+SUBPSEED 717           int   random seed used to generate `SUBPRIORITY` values
+======== ============= ===== ==================================
+
+Required Data Table Columns
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+================================= =========== ================ ===================
+Name                              Type        Units            Description
+================================= =========== ================ ===================
+RA                                float64     deg              Right ascension
+DEC                               float64     deg              Declination
+PMRA                              float32     mas / yr         Proper motion in the RA direction
+PMDEC                             float32     mas / yr         Proper motion in the Dec direction
+REF_EPOCH                         float32     yr               Astrometric reference epoch. Defaults to 2015.5.
+OVERRIDE                          bool                         If ``True`` do not match to and accept an existing primary target. Instead, always generate a new ``TARGETID``.
+FLUX_G                            float32     nanomaggies      `LS`_ flux from tractor input (g)
```

Comment:
Have you verified that unit is in fact `nanomaggies`?  I've discovered that at least some intermediate files used by fiberassign have `nanomaggy` instead.

Reply:
From @araichoor :
```
fitsheader -e 1 $DESI_TARGET/catalogs/dr9/1.1.1/targets/main/resolve/dark/targets-dark-hp-0.fits
=>
...
TTYPE12 = 'FLUX_G  '           / label for field  12                            
TFORM12 = 'E       '           / data format of field: 4-byte REAL              
TUNIT12 = 'nanomaggy'          / physical unit of field                         
...
```

Reply:
I suggest we punt this to a separate issue so that we can merge this PR quickly.  Both 'maggy' and 'maggie' are accepted by the current data model verification code.

Reply:
Actually, @geordie666, if you could concentrate on the veto.fits file, I will go ahead and update the regular expression and search for duplicate titles.

Reply:
I have completed updating the regex for BITNAME files as well as deduplicating the titles.

Reply:
@weaverba137: I've added a description of the veto file (e.g., https://data.desi.lbl.gov/desi/users/adamyers/html/DESI_TARGET/SCND_DIR/PHASE/index.html). The file is consistent with the rest of the data model, except for the lower-case filename, but it has an unusual origin, so I think it was wise to document it.

I see you've assigned me other issues, but if you feel the veto file is now adequately described, feel free to merge this PR and I will work on those other issues tomorrow.

Reply:
Yes, this is ready for merge.  I'll do that momentarily.